### PR TITLE
Add isPodcast indicator

### PR DESCRIPTION
**Pull request by debugwand**
ES Interface isPodcast indicator:
https://github.com/Financial-Times/next-es-interface/blob/8982c96c901dd9e95df5846e27297defc7211395/server/transformers/plugins/audio/teaser.js#L15

### CODE REVIEW

File: components/x-teaser/Props.d.ts [hunk 1]
```diff
@@ -133,6 +133,7 @@ export interface Indicators {
 	accessLevel: 'premium' | 'subscribed' | 'registered' | 'free';
 	isOpinion?: boolean;
 	isColumn?: boolean;
+	isPodcast?: boolean;
```
Comment (i-like-robots) [inline components/x-teaser/Props.d.ts]: Does this mean the top level `type` property will be `audio` in future rather than `podcast`?

Comment (debugwand) [reply components/x-teaser/Props.d.ts]: yes. type audio, subtype podcast, isPodcast in the teaser indicators.

Comment (i-like-robots) [reply components/x-teaser/Props.d.ts]: Good to know (though I'm sure you've told me this already!)

**Comment by robsquires** [comment]
LGTM 👍  Am I right in thinking we'll be able to add a story for x-teaser once we finalise the design?

**Pull request merged**